siac/walletcmd.go

```diff
@@ -224,8 +227,11 @@ func walletloadseedcmd() {
 }
```

Comment:
I think it's implying "[The] Wallet loading [was] successful," but "Loaded 0.3.3.x wallet [filename]" would be more clear.